themes/src/main/resources/theme/base/login/login-config-totp.ftl

```diff
@@ -57,7 +57,7 @@
                 </div>
                 <div class="${properties.kcInputWrapperClass!}">
                     <input type="text" id="totp" name="totp" autocomplete="off" class="${properties.kcInputClass!}"
-                           aria-invalid="<#if messagesPerField.existsError('totp')>true</#if>"
+                           aria-invalid="<#if messagesPerField.existsError('totp')>true</#if>" dir="ltr"
```

Comment:
Adding `dir` attributes selectively here and there seems like a very error prone way to do this, as it is likely to get our of sync of simply be forgotten. Would setting this globally not be enough?

Reply:
@jonkoops username and password are mostly in English, so that's why I chose to make those text fields always LTR.